samples/SocialSample/Startup.cs

```diff
@@ -135,16 +135,16 @@ 127.0.0.1 MsSecSample.localhost.this
                 options.ClaimsIssuer = "OAuth2-Github";
                 options.SaveTokensAsClaims = false;
                 // Retrieving user information is unique to each provider.
-                options.Notifications = new OAuthAuthenticationNotifications
+                options.Events = new OAuthAuthenticationEvents
```

Comment:
Well it still feels clunky to me, as we have Authentication in the namespace, it seems redundant to put it in every class name as well.  Its consistently too long :)